frontends/web/src/components/devices/bitbox02/bitbox02.tsx

```diff
@@ -605,7 +610,7 @@ class BitBox02 extends Component<Props, State> {
                                         <div className={style.stepContext}>
                                             <p>{t('bitbox02Wizard.stepBackup.createBackup')}</p>
                                             <p className="m-bottom-default">{t('bitbox02Wizard.stepBackup.beforeProceed')}</p>
-                                            <form ref={this.setDisclaimerRef}>
+                                            <form>
                                                 <div className="m-top-quarter">
                                                     <Checkbox onChange={this.handleDisclaimerCheck} className={style.wizardCheckbox} id="agreement1" label={t('bitbox02Wizard.backup.userConfirmation1')} />
```

Comment:
don't you also need to put `checked={agreement1}` here and same with all others? e.g. like described here:

https://reactjs.org/docs/forms.html#handling-multiple-inputs

Reply:
Thanks, yes you are right

Reply:
fixed and force pushed https://github.com/digitalbitbox/bitbox-wallet-app/compare/fd680efa61c2ea9244ce3a6c3646f9b6ecf47704..f20078846e3a01774b7715db466574c38d13395c